tests/test_smoke.py

```diff
@@ -968,6 +968,22 @@ def test_new_bucket_creation_and_deletion(self, tmp_local_storage_obj,
         out = subprocess.check_output(['sky', 'storage', 'ls'])
         assert tmp_local_storage_obj.name not in out.decode('utf-8')
 
```

Comment:
Does this test fully pass? (double check)

Reply:
Test passes S3 case but my GCS project/credentials are buggy so might need validation on your end.

Reply:
Try to fix the GCS credentials on your end.

Reply:
We discussed more on this, need to reset gcloud on local laptop.